bin/install-sdk

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Error codes + association:
+# 1 - api.github.com request failed
+# 2 - Invalid argument(s)
+# 3 - Requested SDK DNE
+# 4 - No $THEOS
+# 5 - Requested SDK already exists
+# 6 - Failed curl
+
+set -e
+
+urls=$(curl -s https://api.github.com/repos/theos/sdks/releases/latest | grep download_url | sed 's/.*: "\(.*\)"/\1/')
+if [[ -z "$urls" ]]; then
+	echo "ERROR: api.github.com request failed?!" >&2
+	exit 1
+fi
+
+sdks=( $(echo "$urls" | sed -e 's/.*\///' -e 's/\.sdk.*//' | tr '\n' ' ') )
+sdks+="latest latest-tv"
```

Comment:
Seems like this is behaving differently on different versions of `bash`.

With `GNU bash, version 3.2.57(1)-release (arm64-apple-darwin23)` I'm seeing this append result in `AppleTVOS10.2latest latest-tv AppleTVOS11.4 AppleTVOS12.4 ...` (i.e. the string is being appended to the first element of the array).

It looks like this should be

```suggestion
sdks+=( "latest" "latest-tv" )
```

Reply:
Thanks for catching this!